### PR TITLE
make uptime check period updatable

### PR DESCRIPTION
**Pull request by DrFaust92**
If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [x] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
monitoring: added update support for `period` in `google_monitoring_uptime_check_config`
```

### CODE REVIEW

File: mmv1/third_party/terraform/tests/resource_monitoring_uptime_check_config_test.go [hunk 1]
```diff
@@ -114,11 +114,12 @@ func TestAccMonitoringUptimeCheckConfig_jsonPathUpdate(t *testing.T) {
 	})
 }
 
-func testAccMonitoringUptimeCheckConfig_update(suffix, path, pwd, project, host string) string {
+func testAccMonitoringUptimeCheckConfig_update(suffix, perdiod, path, pwd, project, host string) string {
```
Comment (slevenick) [inline mmv1/third_party/terraform/tests/resource_monitoring_uptime_check_config_test.go:117]:
```suggestion
func testAccMonitoringUptimeCheckConfig_update(suffix, period, path, pwd, project, host string) string {
```

**Comment by DrFaust92** [comment]
[slevenick](https://github.com/slevenick) gentle reminder

**Review by slevenick — CHANGES_REQUESTED** [review]
Failing with: 
`google-beta/resource_monitoring_uptime_check_config_test.go:120:9: fmt.Sprintf format %s reads arg #6, but call has 5 args`

**Review by slevenick — CHANGES_REQUESTED** [review]
I'm not entirely sure why but it looks like there is a syntax error in the produced config:

          on terraform_plugin_test.tf line 5, in resource "google_monitoring_uptime_check_config" "http":
           5:   period       = 60s

**Comment by DrFaust92** [comment]
[slevenick](https://github.com/slevenick) fixed but not able to test locally for some reason

**Pull request merged**